jobs/.env

```diff
@@ -2,5 +2,6 @@ PIPELINES_URI = "http://127.0.0.1:8001"
 ASSETS_URI = "http://127.0.0.1:8000"
 ANNOTATION_MICROSERVICE_URI = "http://127.0.0.1:8002"
 TAXONOMY_URI = "http://127.0.0.1:8002"
+USERS_HOST = "http://dev2.badgerdoc.com/api/v1/users"
```

Comment:
Is it good idea to hardcode external URL here? Why not to use internal, like `TAXONOMY_URI`?

Reply:
Anyway, this value is rewritten in deployment.yaml chart.
I am deleting this line in order to avoid any confusion